experimental/regression_suite/shark-test-suite-models/sdxl/test_unet.py

```diff
@@ -176,7 +287,7 @@ def test_compile_unet_pipeline_rocm(sdxl_unet_pipeline_mlir):
 def test_compile_unet_rocm(sdxl_unet_mlir):
     VmfbManager.sdxl_unet_rocm_vmfb = iree_compile(
         sdxl_unet_mlir,
-        ROCM_COMPILE_FLAGS,
+        ROCM_COMPILE_FLAGS + FP16_UNET_FLAGS,
```

Comment:
Now that there are several variations, please put the data types in all test names, and generated artifact names.